### ISSUE TITLE
Edit Currencies (under General Data?)

### ISSUE THREAD

**Issue by metal450** [portfolio-performance/portfolio#1633]
Apologies if this is already mentioned somewhere in German - I tried searching both github & the forum, to no avail.

Under General Data/Currencies, is there some way to edit the list? I can't figure out any way to remove nor add any different currency pairs. Right-click doesn't bring up a context menu, & I wasn't able to find anything in the preferences.

Thanks!

**Comment by chrisaut** [comment]
No, this is not possible currently. There have been a few proposals to add custom currencies along with sources to retrieve exchange rates, but my understanding is it's rather low on the priority list as most major currencies are already built in.

Which currency are you missing?

**Comment by metal450** [comment]
The issue is that the base currency for all of them is EUR, whereas my currency is USD.

**Comment by chrisaut** [comment]
Other than the display of exchange rates under currencies, does this block you from doing anything?
If your account/portfolio is setup to trade in USD, it should still calculate even if it has to go through EUR as an intermediate (eg. CHF -> EUR -> USD), although it is probably not as quite accurate as a direct exchange rate to the USD. Is that what you mean?

**Comment by metal450** [comment]
Nope, it doesn't block me from doing anything.  I just thought it was a cool feature to be able to reference historical exchange rate charts on that screen (which I've typically done by manually visiting xe.com).  Thought it was nice to be able to do it directly in the software - but I can't, of course, if all the charts are from a different base currency.

**Comment by curbaczek** [comment]
As described in [issue 2382](https://github.com/buchen/portfolio/issues/2382) this feature would be great to work with crypto currencies and the feature request has 3 supporters. I don't see any problems if you just allow users to edit currencies on their own risk. Do I miss anything?
